Pipfile

```
@@ -11,7 +11,7 @@ ipython = "*"
 packaging = "*"
 pre-commit = "*"
 pyjwt = "*"
-pylint = "*"
+pylint = "<2.13"
```

```suggestion
pylint = "*"
```
Since this is an issue of https://github.com/tartley/colorama/issues/86 which was already fixed upstream (see also https://github.com/PyCQA/pylint/issues/6012), I believe it won't take long until a new release is published. Probably, it's sufficient to keep the current Pipfile.lock as it is and remove the version constraint in the Pipfile so we won't forget about it in the future. Hopefully, when we update the dependencies the next time, this issue will no longer exist.